### ISSUE TITLE
syntax.fm site turns blank/black with some content blockers enabled

### ISSUE THREAD

**Issue by patricknelson** [syntaxfm/website#1563]
The site appears normally for an instantant, but immediately turns to a blank/black page on iPadOS (17.2) Safari. To be fair, I have content blockers installed (BlockBear and NeverAds). Disabling those blockers allows the site to work normally. This was not an issue before (not sure when this started). 

Suggested solution to either maybe detect this situation and gracefully degrade or request that users disable their ad blockers (if the functionality that is required that is being blocked is essential for whatever reason). 😊 

<img width="1194" alt="image" src="https://github.com/syntaxfm/website/assets/4269377/203f3361-324a-4968-8f01-3f00a69cb837">

**Comment by wesbos** [comment]
Yes! Thank you! Related to this but we haven't been able to replicate it. Will look into this. https://github.com/syntaxfm/website/issues/1527

Are you able to see if any network requests fail in the dev tools? Normally sentry would help here but you are also blocking sentry 😆

**Comment by patricknelson** [comment]
That's another issue for me as well. 😕 I'm primarily a Windows user on desktop, so debugging safari on my iPad thanks to Apple's closed ecosystem is incredibly difficult, at least for errors that occur immediately on page load (thankfully there's [this amazing bookmarklet](https://paul.kinlan.me/quick-console-bookmarklet-for-desktop-and-mobile/) for showing errors _post_ init).

I recall an old hacky workaround that required running a Node.js server, installing iTunes (garbage software on Windows) and some other stuff. I see something else now which looks similar, note Node.js though (maybe written in C). https://github.com/google/ios-webkit-debug-proxy. Getting all that setup on my company Windows comp might not be feasible.

Luckily I have a test Macbook pro that I never use, so I'll have to login to that when I can (it's locked, gotta reset it) so I'll try to see if I can get into that again. Haven't opened it since we last rotated PW's >4mo ago 😬

**Comment by wesbos** [comment]
I got it to work, BlockBear turned on caused it. Debugging..

**Comment by wesbos** [comment]
It doesn't like that the module is called ShareButton 

<img width="1767" alt="Screenshot 2024-02-20 at 3 53 38 PM" src="https://github.com/syntaxfm/website/assets/176013/1546871d-b318-428e-8df7-44534a7bfc3f">

**Comment by wesbos** [comment]
Fixed!

**Comment by patricknelson** [comment]
Awesome! Meanwhile I was still waiting on IT to get back to me. Thanks for tackling!
